### PR TITLE
Bump Gradle Enterprise Gradle plugin to 3.13

### PR DESCRIPTION
**Pull request by runningcode**
in `gradle.sh` script

### CODE REVIEW

File: components/scripts/lib/gradle.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@ invoke_gradle() {
 
   if [ "$enable_ge" == "on" ]; then
     args+=("-Dcom.gradle.enterprise.build-validation.gradle.plugin-repository.url=https://plugins.gradle.org/m2")
-    args+=("-Dcom.gradle.enterprise.build-validation.gradle-enterprise.plugin.version=3.12.6")
+    args+=("-Dcom.gradle.enterprise.build-validation.gradle-enterprise.plugin.version=3.13")
```
[the remark this answers is not in the document]
Comment (etiennestuder) [reply components/scripts/lib/gradle.sh:19]: Fyi, making the injected plugin/extension version configurable is already on the list and almost ready (just the user config part is missing).

Comment (clayburn) [inline components/scripts/lib/gradle.sh:19]: `3.13` is no longer the most recent version (`3.13.1` as of today).

**Review by clayburn — CHANGES_REQUESTED** [review]
This is now out of date due to the release of Gradle Enterprise Gradle Plugin.

I'm going to mark this with `Request changes` just so it doesn't get inadvertently merged. It sounds like this PR may wind up not being merged anyways.

**Comment by runningcode** [comment]
Closing in favor of: https://github.com/gradle/gradle-enterprise-build-validation-scripts/pull/439

**Pull request closed without merging**